### PR TITLE
Q1 Jira Workflow Updates

### PR DESCRIPTION
**Pull request by slokie-so**
## Description

Introduces Community PR Ticketing,  allowing the workflow to confirm the organization prior to creating a ticket.  Updates to Jira Issue Ticketing,  resulting in a confirmation workflow after creation to ensure there is enough information collected prior to ticketing to Jira. Updates the incorrect link to the corrected HTML link

## Motivation and Context

Reduce incomplete information for Bug / Feature Requests,  introduce Community PR tickets to allow Engineering to review and merge community PR with normalized workflow

## How Has This Been Tested?

Will need to test live, since self-hosted runner is the only permitted worker for this workflow

## Types of changes

-   [ ] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [X] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: .github/workflows/jira-issue-transfer.yml [hunk 1]
```diff
@@ -16,12 +16,13 @@
...
 on:
-  issues:
+  label:
     types: 
-      - opened
+      - edited
 jobs:
   build:
     runs-on: self-hosted
+    if: github.event.label.name == 'bug' && github.event.label.name == 'Ticketed'
```
Comment (sircodemane) [inline .github/workflows/jira-issue-transfer.yml:25]: Would this ever actually be true? This looks like the logic requires the field to be two different values at the same time

**Review by sircodemane — CHANGES_REQUESTED** [review]
I have a concern that this is vulnerable to abuse by purposely toggling labels on and off to create duplicate tickets. I don't see a mechanism here for managing that as the code exists currently.

File: .github/workflows/jira-issue-transfer.yml (hunk 1 above)
Comment (slokie-so) [reply .github/workflows/jira-issue-transfer.yml]: Updated commit - feel free to review

**Review by sircodemane — APPROVED** [review]
Lgtm, i think anything beyond this is simply testing how well the action works

**Pull request merged**